ts/Series/Gauge/GaugeSeries.ts

```diff
@@ -0,0 +1,735 @@
+             * @since   2.3.0
+             * @product highcharts
+             */
+            pivot: {},
+
+            /**
+             * The pixel radius of the pivot.
+             *
+             * @sample {highcharts} highcharts/plotoptions/gauge-pivot/
+             *         Pivot options demonstrated
+             *
+             * @type      {number}
+             * @default   5
+             * @since     2.3.0
+             * @product   highcharts
+             * @apioption plotOptions.gauge.pivot.radius
+             */
+
+            /**
+             * The border or stroke width of the pivot.
+             *
+             * @sample {highcharts} highcharts/plotoptions/gauge-pivot/
+             *         Pivot options demonstrated
+             *
+             * @type      {number}
+             * @default   0
+             * @since     2.3.0
+             * @product   highcharts
+             * @apioption plotOptions.gauge.pivot.borderWidth
+             */
+
+            /**
+             * The border or stroke color of the pivot. In able to change this,
+             * the borderWidth must also be set to something other than the
+             * default 0.
+             *
+             * @sample {highcharts} highcharts/plotoptions/gauge-pivot/
+             *         Pivot options demonstrated
+             *
+             * @type      {Highcharts.ColorString|Highcharts.GradientColorObject|Highcharts.PatternObject}
+             * @default   #cccccc
+             * @since     2.3.0
+             * @product   highcharts
+             * @apioption plotOptions.gauge.pivot.borderColor
+             */
+
+            /**
+             * The background color or fill of the pivot.
+             *
+             * @sample {highcharts} highcharts/plotoptions/gauge-pivot/
+             *         Pivot options demonstrated
+             *
+             * @type      {Highcharts.ColorString|Highcharts.GradientColorObject|Highcharts.PatternObject}
+             * @default   #000000
+             * @since     2.3.0
+             * @product   highcharts
+             * @apioption plotOptions.gauge.pivot.backgroundColor
+             */
+
+            tooltip: {
+                headerFormat: ''
+            },
+
+            /**
+             * Whether to display this particular series or series type in the
+             * legend. Defaults to false for gauge series.
+             *
+             * @since   2.3.0
+             * @product highcharts
+             */
+            showInLegend: false
+
+            // Prototype members
+        } as GaugeSeriesOptions);
+
+    /* *
+     *
+     *  Properties
+     *
+     * */
+
+    public data: Array<GaugePoint> = void 0 as any;
+    public points: Array<GaugePoint> = void 0 as any;
+    public options: GaugeSeriesOptions = void 0 as any;
+
+    public yAxis: RadialAxis = void 0 as any;
+    public pivot?: SVGElement;
+    public fixedBox: boolean = true;
+    public forceDL: boolean = true;
```

Comment:
`fixedBox` and `forceDL` are prototype properties (set in extend call) and need to be moved to the interface.